os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/wdt_api.c

```diff
@@ -121,8 +121,7 @@ void watchdog_start(void)
  */
 void watchdog_stop(void)
 {
-	DBG_8195A("Ameba-D2 not support watchdog_stop function!\n");
-	assert_param(0);
+	dbg("Ameba-D2 not support watchdog_stop function!\n");
```

Comment:
@ewoodev Could you check that dbg is ok here or not?

Reply:
yes, It seems like a good idea to always output watchdog_stop using dbg since it is not called under normal circumstances.